ticketguardian/abstract/lazy_load_list.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, cls, size=None, data=None, slice_ind=0, *ext, **params):
         """
         self._cls = cls
         self._size = size or self._cls().get_resource_count(*ext, **params)
-        self._data = data or defaultdict(lambda: None)
+        self._data = params.pop('data', defaultdict(lambda: None))
```

Comment:
Can you elaborate why you're using a defaultdict with the default value None, as opposed to a standard dict?

Reply:
Thats a good point. It looks like it is unnecessary to use a default dict for this with how I am using it. I changed it to a standard `dict`.